cookbooks/tian_parameterization_kinematic_slab/doc/tian_parameterization_kinematic_slab.md

```diff
@@ -0,0 +1,111 @@
+the left boundary become hydrated, carrying 3 wt% water in sediment, 2 wt% water in MORB, and 1 wt% water in gabbro.
+The boundary velocities are all kinematically prescribed, for 0 <= y <= 20km, the velocity is set to 5 cm/yr, and
+for y > 20km the velocity linearly decreases from 5 cm/yr at the slab surface to 2 cm/yr at the top boundary. A
+schematic diagram showing where the model is in the context of a generic subduction zone is shown in {numref}`fig:schematic-diagram-overview`, and a schematic diagram of the model setup is shown in {numref}`fig:schematic-diagram-model`
+
+:::{note}
+The low fluid viscosity and high contrast in density between the solid and fluid phases leads to large fluid
+velocities on the order of m/yr. Combined with the need for high resolution around the fluid to resolve compaction
+viscosity gradients means that this cookbook requires more computational power than some other cookbooks within
+ASPECT.
+:::
+
+```{figure-md} fig:schematic-diagram-overview
+<img src="schematic-diagram-overview.png" style="width:90.0%" />
+
+Schematic diagram showing the model region of a subduction zone, and the actual schematic diagram of the model.
+```
+
+```{figure-md} fig:schematic-diagram-model
+<img src="schematic-diagram-model.png" style="width:90.0%" />
+
+Schematic diagram showing the model design.
+```
+
+## The input file
+This cookbook includes two input files, one which advects the porosity (free fluid) compositional field through the
+darcy velocity, and one which advects the porosity compositional field according to the coupled two-phase reactions
+solved for in the Reactive Fluid Transport model. Both models feature dehydration and rehydration reactions using
+the Tian parameterization and both input files can be found at [cookbooks/tian_parameterization_kinematic_slab](https://www.github.com/geodynamics/aspect/blob/main/cookbooks/tian_parameterization_kinematic_slab/). One
+important problem in models that track the partitioning of fluid into/out of a solid phase is that these
+interactions can be much faster than the time step of the model. To model these type of processes, ASPECT uses
+operator splitting (see also {ref}`sec:benchmark:operator-splitting`): Reactions are solved on a different time
+scale than advection. For this model, this means that at the beginning of each time step, all hydration/dehydration
+reactions are solved using several shorter sub-time steps. In the input file, we have to choose both the size of
+these sub-time steps and the rate (or characteristic time scale) of the solid-fluid reactions, and they have to be
+consistent in the sense that the operator splitting time step can not be larger than the reaction time scale. We
+choose a conservative value for the fluid-solid reaction time scale of 50 kyr for model stability and computational
+efficiency. For a production model one would likely want to reduce this value to be as low as possible without
+affecting model convergence.
```

Comment:
Maybe change it to "...would lead to very large changes in porosity that make the nonlinear solver converge less reliably unless much smaller advection time steps/higher resolution are enforced."